doc/cprover-manual/cbmc-tutorial.md

```diff
@@ -130,6 +130,16 @@ access. If you add a branch to the example that requires that `argc>=3`,
 the bug is fixed and CBMC will report that the proofs of all properties
 have been successful.
 
+To simplify further processing of counterexample traces, CBMC supports XML as a
+possible output format.
+
+```
+    cbmc file1.c --trace --xml-ui
+```
+
+The specification of the XML trace output can be found here:
+[XML Specification](https://raw.githubusercontent.com/diffblue/cbmc/develop/doc/assets/xml_spec.tex)
```

Comment:
This should contain information on how the file is intended to be processed. I think this should work with most `LaTeX` backends with `-shell-escape`?

Reply:
Also, maybe even consider producing it as part of our documentation somehow? The LaTeX itself isn’t super human readable unprocessed.

Reply:
Yeah :no_entry_sign: there must be either a script or a readme next to it explaining how to build it.